backend/tournesol/models/comparisons.py

```diff
@@ -196,18 +190,6 @@ def sample_video(username, only_rated=False):
             video = Comparison.sample_video_to_rate(username)
         return video
 
-    def weights_vector(self):
-        """How important are the individual scores, according to the rater?"""
-        return self._features_as_vector_fcn(suffix="_weight")
-
-    def save(self, *args, **kwargs):
-        """Save the object data."""
-        if not kwargs.pop("ignore_lastedit", False):
-            self.datetime_lastedit = timezone.now()
```

Comment:
This is no longer necessary, since `auto_now` has been set on the field `datetime_lastedit` in https://github.com/tournesol-app/tournesol-backend/pull/55